packages/styles/source/element/text/rich.ts

```diff
@@ -375,6 +375,13 @@ export const advancedDark: JssObject = create.jss.objectWithClassName({
   ...advanced,
   ...themeDarkColors,
 
+  '& table': {
+    ...table.inline,
+    '& tr:nth-child(even)': {
+      background: 'none',
```

Comment:
You may be wondering why use `background` instead of `backgroundColor` 
<img width="242" alt="image" src="https://github.com/user-attachments/assets/c0149bf5-7d04-4fd2-b362-23c06fd46bff" />

Because I needed the same property names in order to override.

It was better to use `none` instead of specifying a color because the "odd" `<tr>`s do not have a background color. Best to conform to that.